modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionList.tsx

```diff
@@ -41,10 +43,15 @@ const ActionList = ({
 			]}
 			className="mt-0 p-1"
 			creationMenuItems={[
-				{
-					label: Liferay.Language.get('add-action'),
-					onClick: createFDSAction,
-				},
+				creationMenuItemLabel === 'New Item Action'
+					? {
+							label: Liferay.Language.get('new-item-action'),
+							onClick: createFDSAction,
+					  }
+					: {
+							label: Liferay.Language.get('new-creation-action'),
+							onClick: createFDSAction,
+					  },
```

Comment:
Isn't `creationMenuItemLabel` already set in a prop? Can we simplify this to just 
```js
creationMenuItems={[
    {
        label: creationMenuItemLabel,
        onClick: createFDSAction,
    },
```

Reply:
You're right, I was stuck on the format without realizing what I had haha